app/components/support_interface/course_apply_status_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= render TagComponent.new(
```

Comment:
I'm not sure what the team policy is on component specs but I would imagine they would be good to have in instances where they are doing more than simply displaying html

Reply:
Would the alternative to be to have this in the view’s controller?

Reply:
@tvararu @duncanjbrown Thoughts? Is this component worthy, or should the code lie somewhere else?

Reply:
Matter of taste! I think I'd turn it into a single `if` with duplication in preference to two ternaries which quietly depend on each other?